core/src/test/java/org/frankframework/logging/IbisThrowablePatternConverterTest.java

```diff
@@ -10,12 +10,13 @@
 import org.apache.logging.log4j.core.LogEvent;
 import org.apache.logging.log4j.core.impl.Log4jLogEvent;
 import org.apache.logging.log4j.message.SimpleMessage;
+import org.junit.jupiter.api.Test;
```

Comment:
Volgens de regels die er bij mij instaan hoort er nog een newline _voor_ org.junit. De test dependencies horen gegroepeerd te worden, zie ook .editorconfig:

`$*,|,java.**,|,javax.**,|,jakarta.**,|,com.mockrunner.**,|,org.awaitility.**,org.hamcrest.**,org.junit.**,org.mockito.**,|,org.*,|,com.**,*,|,nl.nn.adapterframework**,|,org.frankframework.**`

(een '|' is een lege regel ertussen)